src/lib/kafka_storage.ts

```diff
@@ -369,8 +369,10 @@ export class Exporter {
     try {
       if (BLOCKCHAIN === 'utxo') {
         await this.sendData(events, 'height', signalRecord);
-      } else if (BLOCKCHAIN === 'receipts' || BLOCKCHAIN === 'eth') {
+      } else if (BLOCKCHAIN === 'receipts') {
         await this.sendData(events, 'transactionHash', signalRecord);
+      } else if (BLOCKCHAIN === 'eth') {
```

Comment:
Since we do not use the Kafka record key for now, storing the transaction id there is redundant and wastes spaces.